Artem_Zhivushko/02.Git.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,373 @@
+# 02.Git.Local
+
+## Homework Assignment 1: Initializing a Local Repository
+
+```powershell
```

Comment:
@ArtZhiv  it is not "powershell" syntax, it is "shell", please replace everywhere